home.css

```
@@ -0,0 +1,226 @@
+@font-face {
+	font-family: 'Pretendard Variable';
+	font-weight: 45 920;
+	font-style: normal;
+	font-display: swap;
+	src: url('./src/PretendardVariable.woff2') format('woff2-variations');
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Pretendard Variable';
+  flex-shrink: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
  margin: 0;
  flex-shrink: 0;
}

body {
  font-family: 'Pretendard Variable', sans-serif;
}
```

전체선택자는 div, span, a, img 태그도 포함합니다. font-family는 적용되지 않아도 되는 속성에도 적용될 수 있어서 body태그에 포함시키는게 좋겠네요.

body 태그에 폰트를 지정하는 건 페이지 전체에 기본 폰트를 설정하는 효과가 있습니다.